src/gui/accountsettings.cpp

```diff
@@ -202,6 +202,7 @@ void AccountSettings::slotOpenAccountWizard()
 void AccountSettings::slotToggleSignInState()
 {
     if (_accountState->isSignedOut()) {
+        _accountState->account()->resetRejectedCertificates();
```

Comment:
I think it'd be better to have this in Account::signIn, otherwise it doesn't happen when you click login on the tray menu!